cartridges/app_custom_pd/cartridge/static/default/css/experience/components/commerceAssets/customImageAndTextCommon.css

```diff
@@ -0,0 +1,23 @@
+.image-heading-container {
+  top: 0%;
+}
+
+.mainbanner-image {
+  height: 18.75rem;
+}
+
+.image-heading-container .mainbanner-sub-text p {
+  text-decoration: none;
+  background-color: rgba(255, 255, 255, 0.5);
+  padding: 0rem 0.4rem;
+  border-radius: 0.2rem;
+  border: 1px solid #000;
```

Comment:
color variable

Reply:
That is a static file and i found a mistake in my gitignore file, these static files shouldn`t be part of the repo